Sources/ApiVideoPlayer/Views/ControlsView.swift

```diff
@@ -335,14 +386,19 @@ extension ControlsView: ActionBarViewDelegate {
     }
 }
 
-extension ControlsView: SubtitleViewDelegate {
-    func languageSelected(locale: Locale?) {
-        if let locale = locale {
-            playerController.setCurrentSubtitleLocale(locale: locale)
+extension ControlsView: SelectableViewDelegate {
+    func newElementSelected(element: Any) {
```

Comment:
Checking element type is not scalable and could lead to bugs in the future.
Instead you could identify view by returning the view in the delegates.